docs/architecture/modernize-desktop/deploying-modern-desktop-applications.md

```diff
@@ -0,0 +1,332 @@
+---
+title: Deploying Modern Desktop Applications
+description: Everything you need to know about deploying for modern desktop applications.
+ms.date: 09/16/2019
+---
+
+# Deploying Modern Desktop Applications
+
+## Introduction
+
+When you develop desktop applications, besides other development aspects, you
+also have to think about how your application is going to be packaged and
+deployed to the final users' machines. The problem with packaging, deployment,
+and installation is that it usually falls under the umbrella of the IT
+professionals who care about different things than developers.
+
+These days we are all familiar with the DevOps concept where developers and IT
+pros work closely to move applications to their production environments. But if
+you have been in the desktop battle for more than 10 years you might have witnessed a following tory. A team of developers works together hard to meet the
+project deadlines. Business people are nervous since they need the system working
+on numerous user's machines to run the company. On the "D-Day", the
+project manager checks with every developer that their code is working well,
+everything is fine, they can ship. Then the package team comes in generating the
+setup for the app, distribute it to every user machine and a set of
+test users run the application. Well, they try, because before showing any UI
+the application throws an exception that says "Method ~ of object ~ failed".
+Panic starts flowing through the air and a brief investigation points to a young
+and tired developer that has introduced a third-party control, that certainly "worked on the dev machine".
+
```

Comment:
The use of D-Day on the previous paragraph makes me a bit nervous. I'd avoid using in our content and would replace it with "release day" or something.